Sources/SkipUI/Skip/skip.yml

```diff
@@ -82,6 +82,7 @@ build:
         - 'api(libs.androidx.appcompat.resources)'
         - 'api(libs.androidx.activity.compose)'
         - 'api(libs.androidx.lifecycle.process)'
+        - 'implementation("androidx.exifinterface:exifinterface:1.4.1")'
```

Comment:
It might be better to instead add a dependency on [coil-core-android](https://mvnrepository.com/artifact/io.coil-kt.coil3/coil-core-android) (which appears to depend on [exifinterface](https://mvnrepository.com/artifact/androidx.exifinterface/exifinterface), and so will bring it in transitively). That way, you can just use `versionRef("coil")` to make keep everything up to date with the rest of the coil dependencies.